site/source/docs/api_reference/Filesystem-API.rst

```diff
@@ -99,6 +99,17 @@ The *IDBFS* file system implements the :js:func:`FS.syncfs` interface, which whe
 
 This is provided to overcome the limitation that browsers do not offer synchronous APIs for persistent storage, and so (by default) all writes exist only temporarily in-memory.
 
+.. _filesystem-api-fsfs:
+
+FSFS
+-----
+
+.. note:: This file system is only for use when running code inside a browser.
+
+The *FSFS* file system implements the :js:func:`FS.syncfs` interface, which when called will persist any operations to the attached ``FileSystemDirectoryHandle``.
+
+This uses the `File System Access API <https://web.dev/file-system-access/>`_, which is currently only supported by Chromium browsers.
```

Comment:
IIUC this would also work with the Origin Private File System (OPFS) as long as you pass it a root `FileSystemDirectoryHandle` obtained via `navigator.storage.getDirectory()`, which is also supported by Safari! 

The OPFS is also mentioned in https://github.com/emscripten-core/emscripten/issues/15277 and https://github.com/emscripten-core/emscripten/issues/12428.

Reply:
That's my understanding too...I'll verify and update the text.

Reply:
Yup, this works too. And it allows the test to run without any user interaction, which is great!